packages/cmf/src/onError.js

```diff
@@ -85,6 +104,11 @@ function prepareObject(originalState) {
 	if (originalState === null) {
 		return null;
 	}
+	if (isEvent(originalState)) {
+		return {
+			type: originalState.type,
```

Comment:
can't we embed the just first one or two level of the event?
it might be enough to proceed to debug.